02/compose.yml

```
@@ -0,0 +1,28 @@
+services:
+  praha-challenge-db:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: praha-challenge-db-02
+    environment:
+      - MYSQL_ROOT_PASSWORD=password
+      - MYSQL_DATABASE=employees
+    ports:
+      - 3306:3306
+    volumes:
+      - ./initdb.d:/docker-entrypoint-initdb.d
+      - praha-challenge-db-data-02:/var/lib/mysql
```

元々は「サンプル用のDocker Image使ってね！」だった気がするので、それに合わせるならこう？

```suggestion
  praha-challenge-db:
    image: genschsa/mysql-employees
    environment:
      - MYSQL_ROOT_PASSWORD=password
    ports:
      - 3306:3306
    volumes:
      - praha-challenge-db-data-02:/var/lib/mysql
```

---

apple siliconだと`genschsa/mysql-employees`が動かないらしいので、これはナシ